_linter/markdown-linter-style.rb

```diff
@@ -0,0 +1,41 @@
+#  Include all rules by default
+all
+
+#  Rule Modifications
+#  ----------------------------------------------------------------------------
+## MD029 - tells linter to allow ordered lists that will increment by 1.  By 
+## default (w/o modification) the linter doesn't accept lists structured like:
+## 1. List item #1
+## 2. List item #2
+## 3. List item #3
+rule 'MD029', :style => :ordered
+
+
+#  Exclude these rules by design
```

Comment:
Sounds good to me.

Fixed in https://github.com/m-lab/m-lab.github.io/pull/80/commits/8324b1f7179a8ed42d03876bdf31b80080af28d8